src/coreComponents/physicsSolvers/multiphysics/LagrangianContactSolver.cpp

```diff
@@ -2404,7 +2404,7 @@ void LagrangianContactSolver::computeFractureStateStatistics( DomainPartition co
                                                               bool printAll ) const
 {
 
-  array1d< localIndex > localCounter( 3 );
+  array1d< globalIndex > localCounter( 3 );
```

Comment:
`mpiWrapper::gather` was sending `localIndex` and receiving `globalIndex`. Don't know how this is supposed to work...but it was causing problems, so I just made the sending type `globalIndex`. Perhaps we should disable send/receive of different types in our MPI wrappers?

Reply:
Yeah, MPI_Gather won't work if send/receive types are of different size. That said, this function should probably use `MpiWrapper::allReduce()` instead.

Reply:
I think this was just copied from the residual norm evaluation which isn't a simple reduction.